ui/admin/app/routes/scopes/scope/targets/index.js

```diff
@@ -147,7 +151,7 @@ export default class ScopesScopeTargetsIndexRoute extends Route {
               },
             },
           },
-          { pushToStore: true, peekDb: true },
+          { peekDb: true },
```

Comment:
Wouldn't we still need `pushToStore` since the original issue was that we weren't updating the sessions properly in the store? If we're unloading now, this `store.query` is unneeded now right?

Reply:
Removal of `pushToStore` was mainly because its  set to `true` by [default](https://github.com/hashicorp/boundary-ui/blob/25f015b179bfa9f017e3b7e730d1adfe03a91616/addons/api/addon/handlers/sqlite-handler.js#L35). Ya ur right that extra query is uneeded now.